### DISCUSSION TITLE
Client Breaks Screensaver

### DISCUSSION THREAD

**Discussion by xjim1** [OpenSageTV/Community#153] [General]
Recently began using the Sage client and discovered it 'permanently' disables the screensaver.
Screensaver works fine until I start the client.  But it won't engage after that, even after exiting Sage.  Screensaver settings(in win10) look normal.  Have to reboot to reset it.  I compared client's screensaver settings to my server and tried different configs, but no difference.

Any ideas?

**Comment by trk2** [comment]
ui/disable_screen_saver_and_pm - true/false, will turn off system screen saver and power management when SageTV is in use if true

**Reply by xjim1** [reply]
Ok I'll have to do some homework on this.  Thanks for the reply.
